src/align_add.cpp

```diff
@@ -18,7 +18,8 @@ void align_add(ChunkStack &cs, Chunk *pc, size_t &max_col)
    size_t min_col;
    Chunk  *prev = Chunk::NullChunkPtr;
 
-   if (pc != nullptr)
+   if (  pc != nullptr
+      && pc->IsNotNullChunk())
```

Comment:
``` if (pc != nullptr)``` was added to prevent SEGV during the transition process of the code improvement. At the end this will simply be ```prev = pc->GetPrev();``` without any ```if```, because Chunk pointers won't be null.
Checking for ```pc->IsNotNullChunk()``` is not wrong, but it is not necessary either. If ```pc``` is a null chunk, ```pc->GetPrev()``` will return a Chunk::NullChunkPtr, with no effect on the current value of ```prev```